pagopa/io-app
# fix: [IOBP-605] Disable IDPay feature without UAT environment active

## Short description
This PR adds the `isPagoPATestEnabled` FF to the ID Pay saga handlers.

## List of changes proposed in this pull request
- Added `isPagoPATestEnabled` to the ID Pay sagas watcher

## How to test
> [!IMPORTANT]
> This PR must be tested in a production environment.

From the **Profile** section, enable **ID Pay** and check that no session refresh modal is displayed.
